spring-batch-docs/asciidoc/readersAndWriters.adoc

```diff
@@ -1022,7 +1023,8 @@ alone.
 
 It is also common for a flat file to contain records that each span multiple lines. To
 handle this situation, a more complex strategy is required. A demonstration of this
-common pattern can be found in <<multiLineRecords>>.
```

Comment:
This was a link without a href. I added a link to the multiLineRecords sample.

Reply:
We usually use variables to figure out the version to which to link. Perhaps we should do that here.

Reply:
I do agree! but I reverted the change that adds the link (I will do it properly with a variable in another pass and check all the links in the documentation)